test/range.test.js

```diff
@@ -24,6 +30,13 @@ it('selects versions with query out of range', () => {
   expect(browserslist('ie 1-12')).toEqual(['ie 11', 'ie 10', 'ie 9', 'ie 8'])
 })
 
+it('selects a range of android browsers', () => {
+  expect(browserslist('android 4.3-37'))
+    .toEqual([
```

Comment:
You can keep `toEqual` on previous line to make method smaller

Reply:
😛I am really not good at formatting without the help of prettier. If you are okay with prettier, I would suggest we leave it as-is. I can introduce prettier in a separate PR.

Reply:
Don't worry, fix the logic and `// eslint-disable` and I will fix the formatting.